dist/client-side-validations.js

```diff
@@ -10,9 +10,11 @@
   (global = global || self, global.ClientSideValidations = factory(global.$));
 }(this, (function ($) { 'use strict';
 
-  $ = $ && $.hasOwnProperty('default') ? $['default'] : $;
+  $ = $ && Object.prototype.hasOwnProperty.call($, 'default') ? $['default'] : $;
```

Comment:
@dvdbng I've switched back to `prototype.hasOwnProperty` because of this.

This conversion is being done by babel